ext/zlib/tests/gzgetss.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+gzgetss — Get line from gz-file pointer and strip HTML tags - function
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br>
+--SKIPIF--
+<?php
+if(!extension_loaded("zlib")){die("skip - ZLIB extension not loaded");}
+?>
+--FILE--
+<?php
+$handle = gzopen('gzgetss.gz', 'r');
```

Comment:
Always use absolute paths in PHPT (otherwise the files may not be found):

    $handle = gzopen(__DIR__ . '/gzgetss.gz', 'r');